packages/database/src/migrations/20210614003043-RemoveUserRewardsTable-modifies-schema.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`DROP TABLE user_reward`);
+};
+
+exports.down = function (db) {
```

Comment:
No down because I would have had to regenerate all the existing rewards, and it'll almost certainly never be used, so haven't bothered